src/main/webapp/scholarx/2020/mentors.json

```diff
@@ -331,7 +331,18 @@
       "GeoBotany"
     ],
     "projects" : [],
-    "socialLinks" : []
+    "socialLinks" : [
+      {
+        "platform" : "facebook",
+        "icon" : "fa-facebook",
+        "url" : "https://www.facebook.com/nishanta.rajakaruna.7"
+      },
+      {
+        "platform" : "website",
+        "icon" : "fa-globe",
```

Comment:
- `<i class="fas fa-globe"></i>`
This is the full HTML line for the globe icon. I think the issue is that `fas` class at the beginning is not the one in the template.